### ISSUE TITLE
Play track does not play when it goes out of FM1

### ISSUE THREAD

**Issue by notlebowski** [opentx/opentx#5969]
In this setup
https://www.justfly.solutions/index.php/full-repository?view=product&id=20:kinetic

I have 

SF1  !FM Play track "thract" !1x

When FM1 deactivates (press elevator down trim for 1s) it plays this track.

If I reset the model and later on press elevator down trim, it will go out of FM1 without playing the track.

Strange enough... If I replace !FM1 with FM0 it works always.

This is only after model reset.
Not after model selection. 

It also doesn't matter if I use !1x or 1x
I'm using !1x because it otherwise plays it after model selection as well

I have spent a lot of time debugging this finding out when it does what.
A previous ticket was closed with false arguments without properly reading/testing the issue.

I didn't make this ticket to get "my model working". I can do this easily by changing it to FM0 which would react similar.

I created this ticket to improve OpenTX because the implementation doesn't follow the implied logic.

Furthermore I discovered different behaviour between "haptic" and the "play SF's" regarding this behaviour.

**Comment by kilrah** [comment]
Duplicate of #5932

Please don't make new issues, just point out what we might have missed from your description.
